api/ccapi/include/model.h

```diff
@@ -73,6 +73,23 @@ for inference and training without any configurations*/
  */
 class Model {
 public:
+  /**
+   * @brief     Statistics from running or training a model
+   */
+  struct RunStats {
```

Comment:
It could in just ml::train::RunStats.

Reply:
Moved to ml::train::RunStats as you recommend